.github/workflows/sidecar-build.yml

```diff
@@ -0,0 +1,42 @@
+---
+name: sidecar-build
+
+on:
+  pull_request:
+    branches: [ '*' ]
+
+jobs:
+  test:
+    name: test and build
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        go: ["1.17"]
+    steps:
+    - uses: actions/setup-go@v2
+      with:
+        go-version: ${{ matrix.go }}
+
+    - uses: actions/checkout@v2
+      with:
+        fetch-depth: 0
+
+    - name: copy kubernetes-csi-addons repo in go src
+      run: |
+        mkdir -p /home/runner/go/src/github.com/csi-addons
+        cp -r /home/runner/work/kubernetes-csi-addons/kubernetes-csi-addons /home/runner/go/src/github.com/csi-addons
+    
+    - name: Set up QEMU
+      uses: docker/setup-qemu-action@v1
+      
+    - name: Set up Docker Buildx
+      id: buildx
+      uses: docker/setup-buildx-action@v1.6.0
+
+    - name: run tests and build docker image
```

Comment:
Just curious, why not use `docker/build-push-action` (with `push: false`) here?

Reply:
I dont think they have multi-arch support https://github.com/marketplace/actions/docker-build-push-action 

Followed exmaple from vol-rep-operator repo.

Reply:
it does support multi-arch: https://github.com/docker/build-push-action/blob/master/docs/advanced/multi-platform.md